src/commands/cmd_list.cc

```diff
@@ -556,14 +556,19 @@ REDIS_REGISTER_COMMANDS(
     MakeCmdAttr<CommandLPushX>("lpushx", -3, "write", 1, 1, 1),
     MakeCmdAttr<CommandRPushX>("rpushx", -3, "write", 1, 1, 1), MakeCmdAttr<CommandLPop>("lpop", -2, "write", 1, 1, 1),
```

Comment:
Could you also move `MakeCmdAttr<CommandLPop>("lpop", -2, "write", 1, 1, 1),` to a new line? And others to have a clear list of commands.

Reply:
https://github.com/infdahai/incubator-kvrocks/blob/aab1cf9784eb9e271408d04c45d991c54b1aa9aa/src/commands/cmd_list.cc#L552-L68

How about this?

Reply:
Yes, it works for me. Personally, I think comments like `// impl the X command here` isn't necessary - it explains nothing and actually is useless.